local-api/formlabs-api-local-openapi.yaml

```diff
@@ -322,6 +331,8 @@ components:
         canonical_model_hash:
           type: string
           description: The hash of the mesh data and its transform (position, orientation, and scale)
+        lock:
```

Comment:
If I understand the description of this PR it might be too late, but I prefer `locking`, the name used throughout PreForm. `lock` gives the impression that this is a boolean 🤷